scripts/handle-user-data

```diff
@@ -100,7 +100,7 @@ prepare_partions() {
     done
 
     BOOT2DOCKER_DATA=`echo "${UNPARTITIONED_HD}1"`
-    mkfs.ext4 -i 8192 -L $LABEL $BOOT2DOCKER_DATA
+    mkfs.xfs -n ftype=1 -i size=2048 -L $LABEL $BOOT2DOCKER_DATA
```

Comment:
I see `-n ftype=1` is important here after reading the documentation here https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.2_release_notes/technology-preview-file_systems

![screenshot from 2018-02-05 21-15-49](https://user-images.githubusercontent.com/1132451/35813497-c5923cc2-0ab9-11e8-891d-9d90408176fd.png)

It would be good to put it as comment
```
XFS file systems must be created with the -n ftype=1 option enabled for use as an overlay.
```

Reply:
Personally I believe this would not be really needed. We also do not explain why we use size=2048 instead of 8192.

Reply:
The default inode size is 256 and from Gluster experience, I remember that if we need to attach more extended attributes to the inode we need to create it with more size. @praveenkumar Any reason you are using 2048?

Reply:
@LalatenduMohanty we were using `8192` in case of ext4 disk creation and that's something not well suited for xfs so I went with the safer option. Even you use 256 bytes per inode that also not sufficient because then SElinux and other bits are not able to accomodate. Why do you think 2048 is something we shouldn't use?